FHEM/00_SIGNALduino.pm

```diff
@@ -2207,6 +2207,19 @@ sub SIGNALduino_Parse_MU($$$$@)
 				%patternList = map { $_ => round($patternListRaw{$_}/$clockabs,1) } keys %patternListRaw; 
 			}
 			
+			my $msgclock;
+			my $clocksource = $ProtocolListSIGNALduino{$id}{clockpos}[0];
+			if (!defined($clocksource))
+			{
+				$clocksource = "";
+			} elsif ($clocksource ne "one" && $clocksource ne "zero") {	# wenn clocksource nicht one oder zero ist, dann wird CP= aus der Nachricht verwendet
```

Comment:
Für welche Werte von $clocksource soll denn dieser Abschnitt ausgeführt werden. $clocksource scheint mir in diesem Code Block nicht verwendet zu werden, demnach könnte da so ziemlich alles oder nichts drinnen stehen?

Reply:
So wie es steht, für alle Werte außer one oder zero, z.B. cp

Reply:
Dann brauchen wir cp nicht als valide Wert definieren oder?

Reply:
Es ist egal ob cp oder CP oder was anderes sinnvolles drinsteht